app/models/auth_service_account.rb

```diff
@@ -0,0 +1,6 @@
+class AuthServiceAccount < Authentication
```

Comment:
Should `AuthServiceAccount < AuthUseridPassword`?  @blomquisg Thoughts?  I'm thinking then most of the changes here wouldn't be needed.